components/blocks/collapse.module.css

```diff
@@ -0,0 +1,40 @@
+.Container {
+  @apply py-4;
+}
+
+.PanelHeading {
+  @apply border border-gray-30 rounded-md p-3;
+}
+
+.PanelTitle {
+  @apply list-none flex items-center mb-0 transition ease-in-out duration-75 hover:opacity-80 cursor-pointer;
+}
+
+.Title {
+  @apply flex-1;
+}
```

Comment:
- When the `<Collapse>` element is expanded, the delineation between the title and rest of the content isn't clear IMO because there isn't a horizontal rule separating them nor a difference in font size / color: 
    ![image](https://user-images.githubusercontent.com/20672874/189620102-5be6df99-db35-4d4f-91cb-af396553a282.png)
- What are you thoughts on slightly increasing the title font size:
```css
.Title {
  @apply flex-1 text-lg;
}
```

![image](https://user-images.githubusercontent.com/20672874/189621105-bf2ad5e6-5111-48cf-9e6c-d72b360de6dd.png)

Reply:
I like it!